components/Movie/index.js

```diff
@@ -14,6 +14,12 @@ const StyledSection = styled.section`
   display: flex;
   justify-content: center;
   margin-bottom: 20px;
+  transition: 0.3s ease-out;
+
+  :hover {
+    transition: 0.3s ease-in-out;
+    transform: translateY(-10px);
```

Comment:
Like this the the transition would only work while you are hovering. It is better to write only the changed properties inside the `:hover` and not the transition it self.